install/kubernetes/helm/istio/values.yaml

```diff
@@ -51,6 +51,11 @@ global:
     excludeInboundPorts: ""
     policy: enabled
 
+    # Sets the value of the "--statsdUdpAddress" proxy argument.
+    # Can be disabled (e.g. when mixer is not installed) by setting this to empty string or removing it.
+    # The default is to use the bridge provided by mixer.
+    envoyStatsd: "istio-statsd-prom-bridge:9125"
```

Comment:
I think we should continue using model when variable for service name should be separate from the port value.

Reply:
Thanks. Make sense.
Like the following?

```yaml
envoyStatsd:
  host: istio-statsd-prom-bridge
  port: 9125
```

Reply:
yep I think this would give us good flexibility.